### DISCUSSION TITLE
Globally shared template variables (Library Variables)

### DISCUSSION THREAD

**Discussion by Darth-Bobo** [grafana/grafana#37101] [Dashboards, Panels & Visualizations]
At present I can set variables to be used within a dashboard, but I find that for a reasonable number I want to use them on several dashboards.

Presently this means defining the variable in every dashboard where I want to use it *and* having to change it in every dashboard if the value has to be altered; inevitably that can lead to the names being slightly different making it difficult to copy graphs between dashboards.

It also means changing the value if I want to copy a dashboard from one installation to another (e.g. change Environment="dev" to Environment="live") and I can see this becoming unmanageable over time.

**Comment by pkolyvas** [comment]
We plan on tackling Global Template Variables (aka Library Variables) in the next couple of quarters. I recommend adding your voice to #1959. 

I've also modified the title to make it easier to find/discover.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We have an issue for this #1959. Feel free to continue the discussion there. Thank you!
